kostyor/upgrades/strategy.py

```diff
@@ -0,0 +1,24 @@
+import abc
```

Comment:
Yes but as part of a pre-upgrade task, we may need to live migrate instances off a host. It is related to this work. So, I'm including it.